modules/apps/data-engine/data-engine-taglib/test/mock/fields.es.js

```diff
@@ -157,6 +157,10 @@ export const FIELDS = [
 		type: 'document_library',
 		value: 'document_library',
 	},
+	{
+		rulesActionDisabled: true,
+		rulesConditionDisabled: true,
+	},
```

Comment:
@rodrigopaulino, can you please add a unit test [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/data-engine/data-engine-taglib/test/js/data-layout-builder/components/rules/editor/Editor.es.js) instead?